Resources/Textures/Objects/Specific/Cargo/mail_bag.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Sprites by Velken, based on the Chem Bag sprites from https://github.com/space-wizards/space-station-14/pull/13965",
```

```suggestion
  "copyright": "Sprites by Velken, based on the Chem Bag sprites from ParadiseSS13/Paradise at https://github.com/ParadiseSS13/Paradise/blob/ebf9a7b5e89f5523c5e1c3d2692bf54dac322cae/icons/obj/chemical.dmi,
```
needs to be the direct link the the original source